Packs/Core/Playbooks/playbook-Office_process_creates_a_scheduled_task_via_file_access.yml

```diff
@@ -0,0 +1,1638 @@
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "74":
+    id: "74"
+    taskid: b0909bb0-2dc0-4caa-8bc0-50b601ed289d
+    type: regular
+    task:
+      id: b0909bb0-2dc0-4caa-8bc0-50b601ed289d
+      version: -1
+      name: Check the initiator process prevalence
+      description: Get the prevalence of a process, identified by process_name.
+      script: '|||core-get-process-analytics-prevalence'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "108"
+    scriptarguments:
+      process_name:
+        simple: ${alert.initiatedby}
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 640,
+          "y": -860
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "84":
+    id: "84"
+    taskid: ba60bd40-48ed-4fb3-8237-7c01baeaa308
+    type: regular
+    task:
+      id: ba60bd40-48ed-4fb3-8237-7c01baeaa308
+      version: -1
+      name: Extract the schedule task path
+      description: Extracts regex data from the provided text. The script support groups and looping.
+      scriptName: MatchRegexV2
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "14"
+    scriptarguments:
+      contextKey:
+        simple: ScheduleTaskPath
+      data:
+        simple: ${alert.filepath}
+      ignore-outputs:
+        simple: "false"
+      regex:
+        simple: (?i)(?<=\\System32\\Tasks\\)(.*)
+    separatecontext: false
+    continueonerror: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 650,
+          "y": 1240
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "86":
+    id: "86"
+    taskid: c0b40698-7a10-41aa-8aa0-52fc6f4d3485
+    type: title
+    task:
+      id: c0b40698-7a10-41aa-8aa0-52fc6f4d3485
+      version: -1
+      name: Terminate Process
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "87"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 180,
+          "y": 1110
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "87":
+    id: "87"
+    taskid: ddc7820a-e843-4449-807f-e81131bd4fdf
+    type: regular
+    task:
+      id: ddc7820a-e843-4449-807f-e81131bd4fdf
+      version: -1
+      name: Terminate Causality (CGO)
+      description: Terminate a process tree by its causality ID. Available only for Cortex XSIAM 2.4.
```

Comment:
We will be releasing Cortex XSIAM 2.5 soon.  Should this say Available for Cortex XSIAM 2.4 and above.

Reply:
Done.